dashboard/src/pages/Capability/Components/ShowComponent/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+
+import { Modal, Space, Table, Tag, Tooltip, Typography } from 'antd';
+
+import { ShowParameters } from './types';
+
+export default ({ name, parameters }: ShowParameters) => {
+  const [localVisible, setLocalVisible] = useState(false);
+
+  const { Paragraph, Text } = Typography;
+
+  useEffect(() => {
+    setLocalVisible(name !== '');
+  }, [name, parameters]);
+
+  return (
+    <Modal
+      forceRender
+      visible={localVisible}
+      onCancel={() => setLocalVisible(false)}
+      footer={null}
+      width={1000}
+      maskClosable
+      destroyOnClose
+    >
+      <h2>{`Type: ${name}`}</h2>
+      <Table
+        columns={[
+          {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+            width: 200,
+            render: (text, row) => [
+              <Paragraph copyable={{ text: row.name }}>
+                <Space size="small" align="center">
+                  {row.name}
+                  {!row.required ? undefined : <Tag color="geekblue">request</Tag>}
+                </Space>
+              </Paragraph>,
+            ],
+          },
+          {
+            title: 'Short',
+            dataIndex: 'short',
+            key: 'short',
+            width: 100,
+            responsive: ['md'],
+            render: (text, row) => [
+              <Space size="small" align="center">
+                {!row.short ? undefined : <Tag color="magenta">{text}</Tag>}
+              </Space>,
+            ],
+          },
+          {
+            title: 'Usage',
+            ellipsis: {
+              showTitle: false,
```

Comment:
used `Tooltip` or set `showTitle: true` ?

<img width="817" src="https://user-images.githubusercontent.com/10176784/103116351-2b981c00-46a1-11eb-8f5a-33247e14c1b6.png">

Reply:
Tooltip is already in use

![image](https://user-images.githubusercontent.com/24563928/103121082-850a4600-46b5-11eb-8873-3fdf0c383c69.png)